<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="map_path" default="$(find graph_tool)/maps/" />
  <arg name="map_name" default="bitvector"/>
  <arg name="isnew" default="false"/>
  <arg name="searchradius" default="0.5"/>

  <node name="graph_tool_node" pkg="graph_tool" type="graph_tool_node" output="screen">
    <param name="map_path" value="$(arg map_path)"/>
    <param name="map_name" value="$(arg map_name)"/>
    <param name="isnew" value="$(arg isnew)"/>
    <param name="searchradius" value="$(arg searchradius)"/>
  </node>

  <!-- has been replaced-->
  <!--node name="line_server" pkg="graph_tool" type="line_server" />
  <node name="deletepoint_server" pkg="graph_tool" type="deletepoint_server"/>
  <node name="deletepath_server" pkg="graph_tool" type="deletepath_server"/-->

  <node name="map_server" pkg="map_server" type="map_server" args="$(find graph_tool)/maps/bitmap.yaml" />

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find graph_tool)/config/test.rviz"/>

  <node pkg="graph_tool" type="pcdviewer" name="pcdviewer" args="$(find graph_tool)/maps/bit_mapsmall.pcd"/>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen"/>

</launch>
